lib/Service/Importer/Systems/TrelloJsonService.php

```diff
@@ -344,7 +344,7 @@ private function translateColor(?string $color): string {
 
 	private function replaceUsernames(string $text): string {
 		foreach ($this->getImportService()->getConfig('uidRelation') as $trello => $nextcloud) {
-			$text = str_replace($trello, $nextcloud->getUID(), $text);
+			$text = str_replace($trello, $nextcloud, $text); 
```

Comment:
Could you remove the trailing space? That is causing the Ci failure on the code style check. I tried to push, but github refuses to let me push to your branch

Reply:
@mulles Your PR is still waiting for this little alignment.

Reply:
@juliushaertl  just removed the trailing space, hope that checks run and is mergeable now. 
Thx for your patience